pkmnhelper/listener.py

```diff
@@ -162,6 +164,9 @@ def __init__(self, bot: commands.Bot) -> None:
             entry = db.get_pokedex_entry(player_id, truename)
             entry.caught = True
             entry.save()
+        await self.clean_last_message(message)
+
+    async def clean_last_message(self, message: discord.Message) -> None:
```

Comment:
We don't actually care about the message itself, so just pass the channel in instead